cranelift/codegen/src/abi.rs

```diff
@@ -119,6 +123,13 @@ pub fn legalize_args<AA: ArgAssigner>(args: &[AbiParam], aa: &mut AA) -> Option<
                 args.to_mut()[argno].location = loc;
                 argno += 1;
             }
+            // Assign argument to a location, change type to `INVALID` and move on to the next one.
```

Comment:
nit: incorrect comment, this changes the type to `ty`

```suggestion
            // Assign argument to a location, change type to the requested one and move on to the next one.
```